jdaviz/container.vue

```diff
@@ -28,18 +25,18 @@
     >
         <div>
           <v-row dense style="background-color: #205f76; margin: 0px" class="jdaviz-viewer-toolbar">
-            <j-viewer-data-select
-              :data_items="data_items" 
-              :viewer="viewer"
-              :app_settings="app_settings"
-              :layer_icons="layer_icons"
-              :icons="icons"
-              :linked_by_wcs="viewer.linked_by_wcs"
-              @data-item-visibility="$emit('data-item-visibility', $event)"
-              @data-item-unload="$emit('data-item-unload', $event)"
-              @data-item-remove="$emit('data-item-remove', $event)"
-              @change-reference-data="$emit('change-reference-data', $event)"
-            ></j-viewer-data-select>
+            <j-tooltip tooltipcontent="data-menu is now opened by clicking on the legend in the top-right of the viewer">
```

Comment:
Is it possible when people click this anyway, it opens up the data menu at the new location (in addition to this tooltip)? Took me a second to understand what "legend in the top-right of viewer" really mean. I suspect it will be confusing to others as well.

Reply:
Hmm... What about popping up a screenshot or putting a link in the tooltip to point to a screenshot of the new location? If not, it is okay.

Reply:
managed to figure out a way to open the data menu (with the caveat that it will open all instances if there are multiple instances of the app visible)

Reply:
Nice!